app/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_size = 2
+indent_style = space
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Contradictory rule?

Reply:
this file was autogenerated...I think it's false just for .md files, true for others. heres what it means, idk how useful it is:
![image](https://github.com/user-attachments/assets/70c47f15-096c-4a22-bbe9-f7c20e68752c)

Reply:
Makes sense, my bad.